packages/eservice-template-process/src/services/validators.ts

```diff
@@ -45,3 +46,16 @@ export function assertIsReceiveTemplate(template: EServiceTemplate): void {
     throw templateNotInReceiveMode(template.id);
   }
 }
+
+export function assertIsDraftEserviceTemplate(
+  eserviceTemplate: EServiceTemplate
+): void {
+  if (
+    eserviceTemplate.versions.length > 0 &&
+    eserviceTemplate.versions.every(
+      (v) => v.state !== eserviceTemplateVersionState.draft
+    )
+  ) {
+    throw eserviceTemplateNotInDraftState(eserviceTemplate.id);
+  }
```

Comment:
Sorry, I think my previous comment was wrong. 
The `any` here was fine, since the e-service template is considered Draft if it ha one version and it is in `Draft` state